### ISSUE TITLE
Move build dependencies to main package.json

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/server#13203]
**Is your feature request related to a problem? Please describe.**
Build dependencies, e.g. for for jsunit are installed in the `build` dir. The way we install and run those scripts is very fragile, handled with a custom shell script.

**Describe the solution you'd like**
* Move build/test dependencies to the main package.json once https://github.com/nextcloud/server/pull/13156 is merged into master.
* Don't kill and re-install node_modules all the time
* Use a npm script to invoke the tests instead of the custom test script

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

cc @nextcloud/javascript

**Comment by ChristophWurst** [comment]
I think this is not what @juliushaertl and others want (can't find the ticket with that statement anymore) and since I don't have a strong preference I'm fine with leaving it separate.
